src/sortStudents.ts

```diff
@@ -1,16 +1,67 @@
 
+function getAvarageGrade(gradeArr: number[]): number {
+  return gradeArr.reduce((prev, item) => {
+    return prev + item;
+  }, 0) / gradeArr.length;
+}
```

Comment:
better rename more meaningful way, for example 👇 
```suggestion
  return gradeArr.reduce((total, grade) => {
    return total + grade;
  }, 0) / gradeArr.length;
}
```